HousingRepairsOnlineApi.Tests/HelpersTests/AppointmentTimeToRepairAvailabilityMapperTests.cs

```diff
@@ -62,6 +62,6 @@ public void GivenAppointmentTimeWithStartAndEndTime_WhenMapping_ThenReturnedRepa
         var actual = systemUnderTest.Map(appointmentTime);
 
         // Assert
-        actual.Display.Should().Be("23 November 2022 between 8:00AM to 12:00PM");
+        actual.Display.Should().Be("23 November 2022 between 8:00am to 12:00pm");
```

Comment:
This test has passed previously during the CI pipeline and probably doesn't need changing.
A failure could this be specific machine - suggests the test needs to change, not just the expected outcome.

Reply:
I think the outcome of AppointmentTimeToRepairAvailabilityMapper.Map, which uses Date.ToString, should use CultureInfo.InvariantCulture as the IFormatProvider to ensure consistency:  

          Display = $"{appointmentTime.StartTime.Date.ToString("dd MMMM yyyy", CultureInfo.InvariantCulture)} between {appointmentTime.StartTime.ToString("h:mmtt", CultureInfo.InvariantCulture)} to {appointmentTime.EndTime.ToString("h:mmtt", CultureInfo.InvariantCulture)}",

However, should it be in this commit? Will discuss tomorrow.